habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -621,6 +653,63 @@ class NavToObjSuccessMeasurementConfig(MeasurementConfig):
     success_angle_dist: float = 0.261799
 
 
+@attr.s(auto_attribs=True, slots=True)
+class DistToPickGoalMeasurementConfig(DistToGoalMeasurementConfig):
+    type: str = "DistToPickGoal"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class DistToPlaceGoalMeasurementConfig(DistToGoalMeasurementConfig):
+    type: str = "DistToPlaceGoal"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class CatNavRotDistToGoalMeasurementConfig(RotDistToGoalMeasurementConfig):
+    type: str = "CatNavRotDistToGoal"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class RotDistToPickGoalMeasurementConfig(CatNavRotDistToGoalMeasurementConfig):
+    type: str = "RotDistToPickGoal"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class RotDistToPlaceGoalMeasurementConfig(
+    CatNavRotDistToGoalMeasurementConfig
+):
+    type: str = "RotDistToPlaceGoal"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class NavToPickSuccMeasurementConfig(NavToPosSuccMeasurementConfig):
+    type: str = "NavToPickSucc"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class NavOrientToPickSuccMeasurementConfig(NavToObjSuccessMeasurementConfig):
+    type: str = "NavOrientToPickSucc"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class NavToPlaceSuccMeasurementConfig(NavToPosSuccMeasurementConfig):
+    type: str = "NavToPlaceSucc"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class NavOrientToPlaceSuccMeasurementConfig(NavToObjSuccessMeasurementConfig):
+    type: str = "NavOrientToPlaceSucc"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class PickNavToPlaceSuccMeasurementConfig(MeasurementConfig):
+    type: str = "PickNavToPlaceSucc"
+
+
+@attr.s(auto_attribs=True, slots=True)
+class PickNavOrientToPlaceSuccMeasurementConfig(MeasurementConfig):
```

Comment:
Same with all these

Reply:
Same as above. The are only for defining config params and the default values they can take